odl/operator/operator.py

```diff
@@ -158,28 +365,27 @@ class Operator(with_metaclass(_OperatorMeta, object)):
     ``super().__init__(dom, ran)`` (Note: add
     ``from builtins import super`` in Python 2) in the ``__init__()``
     method of any subclass, where ``dom`` and ``ran`` are the arguments
-    specifying domain and range of the new
-    operator. In that case, the attributes `Operator.domain` and
-    `Operator.range` are automatically provided by
-    `Operator`.
+    specifying domain and range of the new operator. In that case, the
+    attributes `Operator.domain` and `Operator.range` are automatically
+    provided by the parent class `Operator`.
 
-    In addition, any subclass **must** implement **at least one** of the
-    methods ``_apply()`` and ``_call()``, which are explained in the
-    following.
+    In addition, any subclass **must** implement the private method
+    ``_call()``. It signature determines how it is interpreted:
```

Comment:
Since we only have one `_call` may be we can make it `Operator._call` now so users get a link?